modules/nf-core/taxpasta/standardise/main.nf

```diff
@@ -0,0 +1,36 @@
+process TAXPASTA_STANDARDISE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::taxpasta=0.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/taxpasta:0.1.1--pyhdfd78af_0':
+        'quay.io/biocontainers/taxpasta:0.1.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(profile)
+    path taxonomy
```

Comment:
How does this get passed to the command (or not functional yet?)

Reply:
What do you mean?

Reply:
I suppose like `ext.args = { "--taxonomy $taxonomy" }`

Reply:
I would then instead have then

```
script:
taxonomy_cmd = taxonomy ? "--taxonomy $taxonomy" : ""
```

Reply:
Yes, good point.